code/src/lib/Button.js

```diff
@@ -0,0 +1,90 @@
+import styled from "styled-components/macro"
+
+export const Button = styled.button`
+  width: 2.7em;
+  height: 2.3em;
+  background: rgb(141, 97, 67);
+  box-sizing: content-box;
+  color: #FEF7E5;
+  font-weight: 600;
+
```

Comment:
I see the point of breaking a line here to separate the border styling from the above codes but I am not sure it's necessary.